### ISSUE TITLE
Kibana service won't start

### ISSUE THREAD

**Issue by BioSt0rm** [TravisFSmith/SweetSecurity#6]
Unable to start the kibana service after running SweetSecurity script

sudo service kibana start
Job for kibana.service failed. See 'systemctl status kibana.service' and 'journalctl -xn' for details.

systemctl status kibana.service
 kibana.service - LSB: start and stop kibana
   Loaded: loaded (/etc/init.d/kibana)
   Active: failed (Result: exit-code) since Fri 2016-04-08 05:49:46 UTC; 37s ago
  Process: 26845 ExecStart=/etc/init.d/kibana start (code=exited, status=2)

**Comment by ronaldstoner** [comment]
Not sure if our issues are related, but my kibana also woudn't start on Pi3/Raspian Jesse. I had to change the following line in the kibana /etc/init.d/ script:

`. /etc/init.d/functions `
to
`# . /etc/init.d/functions `
`. /lib/lsb/init-functions `

**Comment by honestabe37** [comment]
I don't know if this is related but I just had an issue where Kibana was failing after a few seconds of a start. The logfile indicated a fatal error with "listen EADDRINUSE 127.0.0.1:80"

The kibana.yml file had the port set to 5601 but what I found was that in the /etc/init.d/kibana script the following line was set under `start()`:

` PID=./bin/kibana -p 80  >/dev/null 2>&1 & echo $! > $PIDFILE`

I changed it to:

`PID=./bin/kibana -p 5601  >/dev/null 2>&1 & echo $! > $PIDFILE`

and Kibana started up without issue.
So if you have anything else on your box using port 80, that might be the issue.


Caveat: I am very new to the Linux world so if I'm restating the obvious, my apologies.
